dataset1.py

```
@@ -0,0 +1,111 @@
+        return len(self.x)
+
+    def __getitem__(self, idx):
+        # self.x[idx]의 사이즈는 현재 (input_days, input_dim)이므로, 이를 (input_days * input_dim)으로 flatten함
+        return torch.flatten(self.x[idx]), self.y[idx]
+
+
+def make_features(in_columns, out_columns, input_days, 
+                  is_training, data_dir='data'):
+
+
+    save_fname = f'all_{start}_{end}.pkl'
```

Ruff (0.12.2)

Undefined name `start`

(F821)